<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>메뉴</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>9</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="Btn_test">
         <property name="text">
          <string>TEST</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Btn_kor">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>KOR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Btn_eng">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>ENG</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Btn_insert">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>추가</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Btn_remove">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>삭제</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Btn_save_kor">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>41</height>
          </size>
         </property>
         <property name="text">
          <string>저장(KOR)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Btn_save_eng">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>저장(ENG)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Btn_search">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>찾기</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Btn_find">
         <property name="text">
          <string>찾기</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="autoScroll">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="table_eng">
      <property name="autoScroll">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
